src/lib/mod.rs

```diff
@@ -51,10 +52,11 @@ impl GameOfLife {
     /// All alive cells will be killed.
     pub fn init_empty(mut self) -> Self {
         for y in 1..self.height - 1 {
-            for x in 1..self.height - 1 {
+            for x in 1..self.width - 1 {
                 self.board[y][x] = false;
             }
         }
+        self.generation = 0u64;
```

Comment:
The same for this literal (`0` instead of `0u64`).

Reply:
Same, can't argue with that.